tasks/main.yaml

```diff
@@ -13,6 +7,19 @@
     - Restart sssd
   tags:
     - sssd_pkg
+  when:
+    ansible_os_family == 'RedHat'
+
+- name: Install sssd package for Gentoo
+  portage:
+    name: "{{ sssd_pkg }}"
+    state: present
+  notify:
+    - Restart sssd
+  tags:
+    - sssd_pkg
+  when:
+    ansible_os_family == 'Gentoo'
```

Comment:
There is no need to create installation blocks per distro. Please use the `package` module instead.